services/auth/token/lambdas/generateToken.js

```diff
@@ -5,21 +5,39 @@ import * as response from '../../../../libs/response';
 import { signToken, verifyToken } from '../../../../libs/token';
 import { throwError } from '@helsingborg-stad/npm-api-error-handling';
 import tokenValidationSchema from '../helpers/schema';
+import { logWarn } from '../../../../libs/logs';
 
 const CONFIG_AUTH_SECRETS = config.auth.secrets;
 const ACCESS_TOKEN_EXPIRES_IN_MINUTES = 20;
 const REFRESH_TOKEN_EXPIRES_IN_MINUTES = 30;
 
-export const main = async event => {
+export const main = async (event, context) => {
+  console.log(event);
   const [parseJsonError, parsedJson] = await to(parseJson(event.body));
+
   if (parseJsonError) {
+    logWarn(
+      'JSON Parse error',
+      context.awsRequestId,
+      'service-auth-token-generateToken-001',
+      parseJsonError
+    );
+
```

Comment:
These function calls to the log function makes it harder to read the overall code, would prefer if these calls could be boiled down to oneliners instead. 

A suggestion here is to strip the static parameters into a object/collection where they can be retrieved and then passed them to the log function.
```JS
const logs = {
  jsonParseError: {
    message: 'JSON parse error',
    requestId: context.awsRequestId,
    errorCode: 'service-auth-token-generateToken-001'
  }
}

logWarn({ ...logs.jsonParseError, customData: parseJsonError })

```

Reply:
From what i can see the last parameter for customData is a bit more dynamic, so that's way in this case i did not declare it in the logs object

Reply:
hmm this is dependent on the above comment about using object as parameters right?

Reply:
Yes, so this feels a bit convoluted. I can't think of any way that would be much cleaner than how it is right now.

Reply:
Ok Ill keep it as is for now.